ansible/roles/containerd/tasks/redhat.yaml

```diff
@@ -39,15 +39,13 @@
   delay: 6
   with_items:
     - tar
-    - container-selinux
   when:
     - ansible_distribution == 'OracleLinux'
 
 - name: install container-selinux rpm package
   yum:
     name: "{{
       'container-selinux' if offline_mode_enabled
```

Comment:
should remove the if condition here? The `When` condition is already taking care of it.